drivers/hmis_external_apis/app/helpers/hmis_external_apis/external_forms_helper.rb

```diff
@@ -0,0 +1,110 @@
+    name ||= name_from_label(legend)
+    render_form_fieldset(legend: legend, required: required) do
+      render partial_path('form/date'), legend: legend, required: required, name: name
+    end
+  end
+
+  def render_form_select(label:, name:, required: false, options:)
+    render partial_path('form/select'), label: label, options: options, name: name, required: required, html_id: next_html_id
+  end
+
+  def render_form_radio_group(legend:, name:, required: false, options:, &block)
+    render_form_fieldset(legend: legend, required: required) do
+      radios = render(partial_path('form/radio_group_options'), options: options, name: name, required: required, html_id: next_html_id)
+      extra = capture(&block) if block
+      safe_join([radios, extra].compact_blank, "\n")
+    end
+  end
+
+  def render_section(title:, &block)
+    tag.section do
+      safe_join(
+        [tag.h2(title, class: 'h4'), capture(&block)],
+        "\n",
+      )
+    end
+  end
+
+  def render_form_actions
+    render partial_path('form/actions')
+  end
+
+  def render_form_fieldset(legend:, required: false, &block)
+    tag.fieldset do
+      safe_join(
+        [tag.legend(legend, class: required && 'required'), capture(&block)],
+        "\n",
+      )
+    end
+  end
+
+  def render_form_group(needs_validation: false, &block)
+    tag.div(capture(&block), class: "form-group #{needs_validation ? 'needs-validation' : nil}")
+  end
+
+  def render_form_checkbox(label:, name: nil, required: false)
+    name ||= name_from_label(legend)
+    render partial_path('form/checkbox'), label: label, name: name, html_id: next_html_id, required: required
+  end
+
+  def render_dependent_block(input_name:, input_value:, &block)
+    content = capture(&block)
+    content_tag(:div, content, class: 'dependent-form-group fade-effect')
+    render partial_path('form/dependent_group'), content: content, html_id: next_html_id, input_name: input_name, input_value: input_value
+  end
+
+  def next_html_id
+    @field_id ||= 0
+    @field_id += 1
+    "field-#{@field_id}"
+  end
+
+  def render_modal(title:, html_id: next_html_id, blocker: false, &block)
+    content = capture(&block)
+    render partial_path('modal'), content: content, html_id: html_id, title: title, blocker: blocker
+  end
+
+  def page_config
+    presign_url = ENV['EXTERNAL_FORM_S3_PRESIGN_URL']
+    recaptcha_key = ENV['EXTERNAL_FORM_CAPTCHA_KEY']
+    if Rails.env.development?
+      presign_url ||= presign_hmis_external_apis_external_form_path
+      # for development on hmis-warehouse.dev.test
+      recaptcha_key ||= '6Ldbm4UpAAAAAAK9h9ujlVDig1nC4DghpOP6WFfQ'
+    end
+
+    # FIXME move away from this hard-coded configuration
+    @page_config ||= HmisExternalApis::ExternalForms::Config.new(
+      site_title: 'Tarrant County Homeless Coalition',
+      # FIXME we should host this logo
+      site_logo_url: 'https://ahomewithhope.org/wp-content/themes/tchc/assets/images/logo.png',
```

Comment:
We have this logo, it is set in the env with `LOGO`

The code we use to get the path in hmis is:
```
    logo = ENV['LOGO']
    if logo.present?
      logo_path = SerializedAsset.exists?(logo) ? SerializedAsset.get_src(logo) : "theme/logo/#{logo}"
    end
```

Reply:
I'd say just use that for now (i.e. assume that the `LOGO` should go on public forms). If we need another var later we can add

Reply:
Thanks! This is much better but it looks like it will break if the warehouse isn't publicly available. I was thinking it should probably be uploaded to the public bucket along with the HTML.